libbeat/docs/release-notes/5.0.0.asciidoc

```diff
@@ -0,0 +1,268 @@
+[[release-notes-5.0.0]]
+=== 5.0.0 Combined Release Notes
+
+The list below covers changes between 5.0.0-alpha1 to 5.0.0-GA.
+
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Run function to start a Beat now returns an error instead of directly exiting. {pull}771[771]
+- The method signature of `HandleFlags()` was changed to allow returning an error {pull}1249[1249]
```

Comment:
I'd leave these two (which are affecting only Beats writers) out, since a Beats user wouldn't know how this change affects them.